src/dmd/backend/dwarf2.d

```diff
@@ -1,5 +1,5 @@
 
-/* Reflects declarations from the Dwarf 3 spec, not the The D Language Foundation
+/* Reflects declarations from the DWARF specification, not the The D Language Foundation
```

Comment:
I think it would be useful to specify which Dwarf versions it supports (e.g Dwarf 3 to 5).